tests/series/test_temporal_ops.py

```diff
@@ -120,11 +120,70 @@ def ts_maker(h):
 
     input_ts = list(map(ts_maker, input))
     s = Series.from_pylist(input_ts).cast(DataType.timestamp(TimeUnit.ms()))
-    days = s.dt.hour()
+    hours = s.dt.hour()
 
-    assert days.datatype() == DataType.uint32()
+    assert hours.datatype() == DataType.uint32()
 
-    assert input == days.to_pylist()
+    assert input == hours.to_pylist()
+
+    s = Series.from_pylist(input_ts).cast(DataType.time(TimeUnit.ns()))
+    hours = s.dt.hour()
+
+    assert hours.datatype() == DataType.uint32()
+
+    assert input == hours.to_pylist()
+
+
+def test_series_timestamp_minute() -> None:
```

Comment:
Could you also add tests for different timezones for your tests? See `test_series_timestamp_day_operation`.